cpp/modules/deck.gl/core/src/shaderlib/project/viewport-uniforms.h

```diff
@@ -50,15 +50,11 @@ struct ViewportUniforms {
   mathgl::UniformVector3<float> coordinateOrigin;
 };
 
-/**
- * Returns uniforms for shaders based on current projection
- * includes: projection matrix suitable for shaders
- *
- * TODO - Ensure this works with any viewport, not just WebMercatorViewports
- *
- * @param viewport -
- * @return {Float32Array} - 4x4 projection matrix that can be used in shaders
- */
+/// \brief Returns uniforms for shaders based on current projection
+/// \note Includes projection matrix suitable for shaders
+/// \param viewport
+/// \return 4x4 projection matrix that can be used in shaders
+// TODO - Ensure this works with any viewport, not just WebMercatorViewports
```

Comment:
The format should match the one mentioned above

Reply:
I don't think there was an attached email in the original comment. What should be done in a case like this?

Reply:
is `// TODO(unknown):` acceptable?

Reply:
If you know who the original author is, you can use their email, if not you'd just use your own. In this case it was @isaacbrodsky